images/pulumi/configs/latest.apko.yaml

```diff
@@ -0,0 +1,69 @@
+contents:
+  repositories:
+    - https://packages.wolfi.dev/os
+  keyring:
+    - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+  packages:
+    - pulumi
+    - pulumi-watch
+    - wolfi-baselayout
+    - ca-certificates-bundle
+    - busybox # Needed for language support
+
+    # for Pulumi Go support
+    - pulumi-language-go
+    - go
+
+    # for Pulumi Dotnet support
+    - pulumi-language-dotnet
+    - dotnet-7
+    - dotnet-7-runtime
+    - dotnet-7-sdk
+    - aspnet-7-runtime
+    - aspnet-7-targeting-pack
+
+    # for Pulumi Python support
+    - pulumi-language-python
+    - python-3.11
+    - py3.11-pip
+    - python-3.11-dev
+
+    # for Pulumi Node.js support
+    - pulumi-language-nodejs
+    - nodejs
+    - nghttp2
+
+    # for Pulumi Java support
+    - pulumi-language-java
+    - glibc-locale-en
+    - openjdk-17-jre
+    - libstdc++
+    - maven
```

Comment:
I'm surprised it needs maven but only the jre version of java. Is it doing builds?  If so it should probalby include the full jdk.